docs/contributing/the_basics.md

```diff
@@ -4,7 +4,7 @@ An overview on contributing to the _Black_ project.
 
 ## Technicalities
 
-Development on the latest version of Python is preferred. As of this writing it's 3.9.
+Development on the latest version of Python is preferred. As of this writing it's 3.11.
```

Comment:
```suggestion
Development on the latest version of Python is preferred.
```
Let's just remove this sentence so we don't need to keep updating it.

Reply:
That works too, haha :)